.github/workflows/publish-native-assets-to-github-releases.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will run tests using node and then publish a package to GitHub Packages when a release is created
+# For more information see: https://help.github.com/actions/language-and-framework-guides/publishing-nodejs-packages
+
+name: Add native binaries to release
+
+on:
+  release:
+    types: [created]
+    
+
+jobs:
+  augment-release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
We support 7, 9, 11 as well. Can we add these?

Reply:
Sure, I have added these versions into the matrix. 

It took a while to get Windows to work correctly with these versions; I had to pin node-gyp@5.1.1 to get this to work (node-gyp@7.0.0 didn't work -- something went wrong with the cache dir. I presume it's not writeable on the runner). Assuming you accept this PR, the next step would be to open an issue to resolve this.

The exact error can be seen here: https://github.com/spmiller/node-oom-heapdump/runs/801717428?check_suite_focus=true

Reply:
Ok, that's clear. 
I would rather be on the latest version, but if that poses issue, we can leave it as is for now.